deployment/conf/supported_apps_w_extensions.json

```diff
@@ -0,0 +1,498 @@
+      ],
+      "id": 6
+    },
+    "Decompress/Unpack": {
+      "title": "Decompress/Unpack",
+      "app": "kb_uploadmethods/unpack_staging_file",
+      "output_type": [
+        null
+      ],
+      "extensions": [
+        "zip",
+        "tar",
+        "tgz",
+        "tar.gz",
+        "7z",
+        "gz",
+        "gzip",
+        "rar"
+      ],
+      "id": 7
+    },
+    "Samples": {
+      "title": "Samples",
+      "app": "sample_uploader/import_samples",
+      "output_type": [
+        "KBaseSets.SampleSet"
+      ],
+      "extensions": [
+        "csv",
+        "xls",
+        "xlsx"
+      ],
+      "id": 8
+    },
+    "Media": {
+      "title": "Media",
+      "app": "kb_uploadmethods/import_tsv_excel_as_media_from_staging",
+      "output_type": [
+        "KBaseBiochem.Media"
+      ],
+      "extensions": [
+        "tsv",
+        "xls",
+        "xlsx"
+      ],
+      "id": 9
+    },
+    "Expression Matrix": {
+      "title": "Expression Matrix",
+      "app": "kb_uploadmethods/import_tsv_as_expression_matrix_from_staging",
+      "output_type": [
+        "KBaseFeatureValues.ExpressionMatrix"
+      ],
+      "extensions": [
+        "tsv"
+      ],
+      "id": 10
+    },
+    "Metabolic Annotations": {
+      "title": "Metabolic Annotations",
+      "app": "MergeMetabolicAnnotations/import_annotations",
+      "output_type": [
+        "KBaseGenomes.Genome"
+      ],
+      "extensions": [
+        "tsv"
+      ],
+      "id": 11
+    },
+    "Bulk Metabolic Annotations": {
+      "title": "Bulk Metabolic Annotations",
+      "app": "MergeMetabolicAnnotations/import_bulk_annotations",
+      "output_type": [
+        "KBaseGenomes.Genome"
+      ],
+      "extensions": [
+        "tsv"
+      ],
+      "id": 12
+    },
+    "FBA Model": {
+      "title": "FBA Model",
+      "app": "kb_uploadmethods/import_file_as_fba_model_from_staging",
+      "output_type": [
+        "KBaseFBA.FBAModel"
+      ],
+      "extensions": [
+        "tsv",
+        "xls",
+        "xlsx",
+        "smbl"
+      ],
+      "id": 13
+    },
+    "Phenotype Set": {
+      "title": "Phenotype Set",
+      "app": "kb_uploadmethods/import_tsv_as_phenotype_set_from_staging",
+      "output_type": [
+        "KBasePhenotypes.PhenotypeSet"
+      ],
+      "extensions": [
+        "tsv"
+      ],
+      "id": 14
+    },
+    "Attribute Mapping": {
+      "title": "Attribute Mapping",
+      "app": "kb_uploadmethods/import_attribute_mapping_from_staging",
+      "output_type": [
+        "KBaseExperiments.AttributeMapping"
+      ],
+      "extensions": [
+        "xls",
+        "xlsx"
+      ],
+      "id": 15
+    },
+    "EscherMap": {
+      "title": "EscherMap",
+      "app": "kb_uploadmethods/import_eschermap_from_staging",
+      "output_type": [
+        "KBaseFBA.EscherMap"
+      ],
+      "extensions": [
+        "json"
+      ],
+      "id": 16
+    }
+  },
+  "types": {
+    "sra": [
+      [
+        0,
+        1
+      ]
```

Comment:
What is the `1`? Since all the entries seem to have a `1`, is it necessary to add it?

Reply:
I felt it was better to be explicit rather than implicit